src/LiveQuery/QueryTools.js

```diff
@@ -242,6 +242,11 @@ function matchesKeyConstraints(object, key, constraints) {
           return false;
         }
         break;
+      case '$eq':
```

Comment:
Isn't this case already handled?

Reply:
Yes. Currently, `{ "field": "value" }` works but `{ "field": { "$eq": "value" } }` does not work for live query. It could actually even be a separate PR.

Reply:
Just supporting what @davimacedo mentioned about `$eq` not working for LiveQuery is true. @dplewis if you recall, I was using `$eq` in ParseSwift and asked you about it here https://github.com/parse-community/parse-server/pull/7113#issuecomment-758064363. I then made a change to ParseSwift to not use `$eq` anymore here https://github.com/parse-community/Parse-Swift/pull/49